Tests/LibSemVer/TestFromStringView.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2023, Gurkirat Singh <tbhaxor@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/StringView.h>
+#include <AK/Tuple.h>
+#include <AK/Vector.h>
+#include <LibSemVer/SemVer.h>
+#include <LibTest/TestCase.h>
+
+TEST_CASE(parsing) // NOLINT
```

Comment:
Removing this will trigger the [readability-function-cognitive-complexity](https://clang.llvm.org/extra/clang-tidy/checks/readability/function-cognitive-complexity.html). 

```
Function '__test_parsing' has cognitive complexity of 420 (threshold 100)
```

I could have updated the .clang-tidy, but that would affect all the sources. So I am using `// NOLINT` to disable for the specific function. [[ref]](https://stackoverflow.com/a/40325333/10362396)

I will add more details, and disable specific warning for better understanding. [[ref]](https://clang.llvm.org/extra/clang-tidy/#suppressing-undesired-diagnostics)

Reply:
We don't care about `clang-tidy`, I think.